mappings/net/minecraft/entity/passive/AbstractVillagerEntity.mapping

```diff
@@ -1,2 +1 @@
 CLASS aqz net/minecraft/entity/passive/AbstractVillagerEntity
-	FIELD e customer Laro;
```

Comment:
Enigma did this automatically, seems the field no longer exists.

Reply:
Interesting, the obf name seems to be `bD` now.